CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Current (in progress)
 
-- Nothing yet
+- Switch to poetry [#2757](https://github.com/opendatateam/udata/pull/2757)
```

Comment:
Cf https://github.com/opendatateam/udata/pull/2757#discussion_r952769565 for changelog details